client/components/Main.js

```diff
@@ -14,7 +16,10 @@ class Main extends React.Component {
   render() {
     return (
       <Router>
-        <div>Welcome To Global Snacker!</div>
+        <div>
+          <Route component={CreateUser} path="/create-user" exact />
+          <Route component={LogIn} path="/create-user" exact />
```

Comment:
Hm, small detail but I think the path should be /login

Reply:
Thank you, I also changed the name of the CreateUser component to CreateAccount to make the naming match up more with the users' perspective.